pages/api/token/temporary.ts

```diff
@@ -106,4 +115,23 @@ const updateTemporaryToken = async (
   );
 };
 
+const sendTemporaryTokenByMail = async (email: string, temporaryToken: string) => {
+  const notifyClient = new NotifyClient(
+    "https://api.notification.canada.ca",
```

Comment:
1- We could have moved the url in .env even though it doesn't change that often.

Reply:
It is a good idea. @Moro-Code @bryan-robitaille do you think it is worth doing now? Is it simple to add a new ENV variable in staging and production?

Reply:
I would honestly say it's not worth doing now. That URL is not going to change and if it does we'll have ample time to replace it with an env variable if it comes to that :)